frame/staking/src/lib.rs

```diff
@@ -356,12 +356,15 @@ pub struct ValidatorPrefs {
 	/// nominators.
 	#[codec(compact)]
 	pub commission: Perbill,
+	/// Max number of nominators they will payout
+	pub max_payout: u16,
```

Comment:
Would change the name to something like `max_payouts` or `max_nominator_payouts`. `max_payout` semantically sounds like it's a number of tokens.